portfolio/src/main/java/com/google/sps/servlets/ServletHelper.java

```diff
@@ -0,0 +1,78 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobInfo;
+import com.google.appengine.api.blobstore.BlobInfoFactory;
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.appengine.api.images.ImagesService;
+import com.google.appengine.api.images.ImagesServiceFactory;
+import com.google.appengine.api.images.ServingUrlOptions;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Class contains the helper method for the servlet*/
+public final class ServletHelper {
+
+  /* The static store service that can be used to all the servlets */
+  protected static final DatastoreService DEFAULT_DATASTORE_SERVICE
+                              = DatastoreServiceFactory.getDatastoreService();
+  protected static final BlobstoreService DEFAULT_BLOBSTORE_SERVICE 
+                              = BlobstoreServiceFactory.getBlobstoreService();
+
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
+  protected static String getParameterWithDefault(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
How about 'getParameterOrDefault'?

Reply:
To provide more information, there is tons of similar discussions on the naming (in map_util) here:
https://screenshot.googleplex.com/AcGTN6CPZEh

Reply:
I checked the screenshot, does it mean it's okay to name it 'withDefault' if we are passing the default value?

Reply:
Use your own decision~